codegens/java-unirest/lib/unirest.js

```diff
@@ -6,19 +6,22 @@ var _ = require('./lodash'),
 const SUPPORTED_METHODS = ['GET', 'POST', 'PUT', 'HEAD', 'PATCH', 'DELETE', 'OPTIONS'];
 
 /**
- * parses request and returns java unirest code snippet 
- * 
- * @param {Object} request - Postman SDK Request Object 
+ * parses request and returns java unirest code snippet
+ *
+ * @param {Object} request - Postman SDK Request Object
  * @param {String} indentString - indentation required for code snippet
- * @param {Object} options 
+ * @param {Object} options
  * @return {String} - java unirest code snippet
  */
 function makeSnippet (request, indentString, options) {
     var snippet = '',
         urlString = encodeURI(request.url.toString());
 
     if (options.requestTimeout > 0) {
-        snippet += `Unirest.setTimeouts(${options.requestTimeout}, 0);\n`;
+        snippet += `Unirest.setTimeouts(0, ${options.requestTimeout});\n`;
+    }
+    else {
+        snippet += 'Unirest.setTimeouts(0, 0);\n';
```

Comment:
isn't the first arg supposed to be the read timeout?

Reply:
```
Unirest.setTimeouts(long connectionTimeout, long socketTimeout);
Connection timeout: The timeout until a connection with the server is established (in milliseconds), and the default value is 10000
Socket timeout: The timeout to receive data (in milliseconds), and the default is 60000.
We can set any of these timeouts to zero to disable the timeout.
```